_stbt/keyboard.py

```diff
@@ -663,6 +663,8 @@ def navigate_to(self, target, page, verify_every_keypress=False, retries=2):
             self.G_ = _strip_shift_transitions(self.G)
 
         assert page, "%s page isn't visible" % type(page).__name__
+        assert page.selection in self.G_, \
+            "page.selection (%r) isn't in the keyboard" % (page.selection,)
```

Comment:
I *think* this is the right exception. This will be a test **failure** (similar to a non-visible `page` in the previous line), vs. the ValueError (a test **error**) if `target` isn't in the keyboard:

https://github.com/stb-tester/stb-tester/blob/b1b9d4b8c991c24cd461561e96fa0354d86cc432/_stbt/keyboard.py#L657-L659

My thinking is that `target` (or `text`) is usually specified explicitly as a literal string in the test script, whereas a bad `page.selection` could be a bug in the device-under-test.